uvm/hier/usr/share/untangle/bin/ut-factory-defaults

```diff
@@ -4,34 +4,73 @@
 # This file should be rewritten as a run-parts script on a /etc/untangle/restore.d directory
 # This would allow us to create plugins for other packages in the restore.d directory
 
+# A simgple "rm -rf *" can fail if the command path exceeds a length.
+remove_path(){
+    path=$1
+
+    if [ -d $path ]; then
+        # Remove all directories under the path
+        for dir in $path/*; do
+            rm -rf $dir
+        done
+    elif [ -f $path ]; then
+        # Remove all files with the same path root.  For example passing
+        # wizard.js removes that file along with files like wizard.js-version-2022-09-19-164126.916.js
+        for file in $path*; do
+            rm -f $file
+        done
+    fi
+}
+
 systemctl stop untangle-vm
 
+#
+# Remove files
+#
 echo "Removing settings and configuration..."
+remove_path /usr/share/untangle/settings
+remove_path /usr/share/untangle/conf/wizard.js
+remove_path /usr/share/untangle/conf/uris.js
+
+# Remove most old logs
+find /var/log -name "*gz" | xargs rm -f
+find /var/log -name "*.1" | xargs rm -f
+find /var/log -name "*.2" | xargs rm -f
+find /var/log -name "*.3" | xargs rm -f
+find /var/log -name "*.4" | xargs rm -f
+find /var/log/uvm/* | xargs rm -f
```

Comment:
Do we need the extra asterisk? I wouldn't think that would affect the output of `find`. I'm not a `find` wizard though by any means.

Reply:
Yep, the wildcard match is required for file matching. Good question!